R/addAlpha.R

```diff
@@ -0,0 +1,234 @@
+#' Estimate alpha diversity indices
+#' 
+#' The function estimates alpha diversity indices optionally using rarefaction,
```

Comment:
Change: "The function estimates" -> "Estimate"

Reply:
Remove the words: "optionally using rarefaction," (this will be explained later in arguments and @details field)